config/rubocop.yml

```diff
@@ -61,3 +61,20 @@ ExtraSpacing:
 # Buggy
 FormatParameterMismatch:
   Enabled: false
+
+# Align if/else blocks with the variable assignment
+EndAlignment:
+  AlignWith: variable
+
+# Use square brackets for literal Array objects
+PercentLiteralDelimiters:
+  PreferredDelimiters:
+    '%':  '{}'
+    '%i': '[]'
+    '%q': ()
+    '%Q': ()
+    '%r': '{}'
+    '%s': ()
+    '%w': '[]'
```

Comment:
Incidentally, I should probably think about some process where every suggestion that I make or receive is checked to see if it is something that Rubocop or something else can test. I need to begin tracking these so I don't forget them.